notebooks/fig1.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "from biom import load_table\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from qiime2 import Artifact, Metadata\n",
+    "from qiime2.plugins import diversity, empress\n",
+    "import scipy.stats as ss\n",
+    "import seaborn as sns\n",
+    "from skbio.tree import TreeNode\n",
+    "import tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Figure 1A - Full EMP Visualization\n",
+    "\n",
+    "This will walkthrough the procedure of generating the EMPire plot of the full EMP dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Clear the output directory and create a fresh one\n",
+    "!rm -rf fig1/output/*\n",
+    "!mkdir -p fig1/output"
```

Comment:
I think this is also missing `!mkdir -p fig1/input` otherwise the next wget commands fail.

Reply:
Ping :)

Reply:
New version of PR includes the pre-computed PCoA in the input folder so we shouldn't have to `mkdir` it I think.